R/R-to-variable.R

```diff
@@ -61,14 +61,32 @@ setMethod("toVariable", "logical", function (x, ...) {
     return(NAToCategory(out, useNA="always"))
 })
 
-categoriesFromLevels <- function (x) {
-    if (anyDuplicated(x)) {
+#' Convert a factor's levels into Crunch categories. 
+#' 
+#' Crunch categorical variables have slightly richer metadata than R's
+#' factor variables. This function generates a list of category data from
+#' a factor's levels which can then be further manipulated in R before being
+#' imported into Crunch.
+#'
+#' @param levels A character vector containing the levels of a factor. Usually
+#' obtained by running \code{levels()}
```

Comment:
We've started transitioning to the roxygen markdown format, which means you don't need `\code`. That can be either in backticks, or you could make it link to the man page for that base function with `[]`, I think like `[base::levels()]`, but I usually have to google "roxygen markdown" to find their vignette to be sure.